src/main/java/org/mariadb/jdbc/internal/protocol/AbstractConnectProtocol.java

```diff
@@ -797,6 +806,112 @@ public int getMinorServerVersion() {
     }
 
     /**
+    * If the connection string returns MariaDB then it must be as specified.
+     *
+    * @return <code>true</code> if server is MariaDB, otherwise <code>false</code>.
+    */
+    public boolean isMariaDbServer() {
+        return getServerVersion().indexOf("MariaDB") != -1;
+    }
+
+    /**
+     * Whether MariaDB server version is consistent with TLSv1.2 support.
+     *
+     * @return <code>true</code> if server version is consistent with TLSv1.2 support, otherwise <code>false</code>
+     */
+    protected boolean isMariaDbCapableTlsv12() {
+
+        // if not mariadb then immediate false
+        if (!isMariaDbServer()) {
+            return false;
+        }
+
+        // minimum of MariaDB 10.0.15 for TLSv1.2 support
+        if (!versionGreaterOrEqual(10, 0, 15)) {
```

Comment:
yes - lol not sure what happened there, I'll fix and submit again for the millionth time..